features-json/css-read-only-write.json

```diff
@@ -0,0 +1,246 @@
+      "30":"y x",
+      "31":"y x",
+      "32":"y x",
+      "33":"y x",
+      "34":"y x",
+      "35":"y x",
+      "36":"y x",
+      "37":"y x",
+      "38":"y x",
+      "39":"y x",
+      "40":"y x",
+      "41":"y x",
+      "42":"y x",
+      "43":"y x",
+      "44":"y x"
+    },
+    "chrome":{
+      "4":"u",
+      "5":"u",
+      "6":"u",
+      "7":"u",
+      "8":"u",
+      "9":"u",
+      "10":"u",
+      "11":"u",
+      "12":"u",
+      "13":"u",
+      "14":"u",
+      "15":"u",
+      "16":"u",
+      "17":"u",
+      "18":"u",
+      "19":"u",
+      "20":"u",
+      "21":"u",
+      "22":"u",
+      "23":"u",
+      "24":"u",
+      "25":"u",
+      "26":"u",
+      "27":"u",
+      "28":"u",
+      "29":"u",
+      "30":"u",
+      "31":"u",
+      "32":"u",
+      "33":"u",
+      "34":"a #1",
+      "35":"a #1",
+      "36":"a #1",
+      "37":"a #1",
+      "38":"a #1",
+      "39":"a #1",
+      "40":"a #1",
+      "41":"a #1",
+      "42":"a #1",
+      "43":"a #1",
+      "44":"a #1",
+      "45":"a #1",
+      "46":"a #1",
+      "47":"a #1",
+      "48":"a #1",
+      "49":"a #1"
+    },
+    "safari":{
+      "3.1":"u",
+      "3.2":"u",
+      "4":"u",
+      "5":"u",
+      "5.1":"u",
+      "6":"u",
+      "6.1":"u",
+      "7":"a #1",
+      "7.1":"a #1",
+      "8":"a #1",
+      "9":"a #1"
+    },
+    "opera":{
+      "9":"n",
+      "9.5-9.6":"n",
+      "10.0-10.1":"n",
+      "10.5":"n",
+      "10.6":"n",
+      "11":"n",
+      "11.1":"n",
+      "11.5":"n",
+      "11.6":"n",
+      "12":"n",
+      "12.1":"n",
+      "15":"a #1",
+      "16":"a #1",
+      "17":"a #1",
+      "18":"a #1",
+      "19":"a #1",
+      "20":"a #1",
+      "21":"a #1",
+      "22":"a #1",
+      "23":"a #1",
+      "24":"a #1",
+      "25":"a #1",
+      "26":"a #1",
+      "27":"a #1",
+      "28":"a #1",
+      "29":"a #1",
+      "30":"a #1",
+      "31":"a #1",
+      "32":"a #1",
+      "33":"a #1",
+      "34":"a #1"
+    },
+    "ios_saf":{
+      "3.2":"u",
+      "4.0-4.1":"u",
+      "4.2-4.3":"u",
+      "5.0-5.1":"u",
+      "6.0-6.1":"u",
+      "7.0-7.1":"u",
+      "8":"a #1",
+      "8.1-8.4":"a #1",
+      "9":"a #1"
+    },
+    "op_mini":{
+      "5.0-8.0":"n"
+    },
+    "android":{
+      "2.1":"n",
+      "2.2":"n",
+      "2.3":"n",
+      "3":"n",
+      "4":"n",
+      "4.1":"n",
+      "4.2-4.3":"n",
+      "4.4":"n",
+      "4.4.3-4.4.4":"n",
+      "44":"a #1"
+    },
+    "bb":{
+      "7":"u",
+      "10":"u"
+    },
+    "op_mob":{
+      "10":"n",
+      "11":"n",
+      "11.1":"n",
+      "11.5":"n",
+      "12":"n",
+      "12.1":"n",
+      "30":"a #1"
+    },
+    "and_chr":{
+      "46":"a #1"
+    },
+    "and_ff":{
+      "41":"y x"
+    },
+    "ie_mob":{
+      "10":"u",
+      "11":"u"
+    },
+    "and_uc":{
+      "9.9":"u"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1":"Suppports selector only for input and textarea fields, but not for contenteditable",
```

Comment:
Trailing comma here is causing the json to be invalid. (in the future I suggest using http://jsonlint.com to check for json errors)